src/dbus/instropect.xml

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="io.openthread.BordeRrouter">
+    <method name="Scan">
```

Comment:
do we want to scan specific channel?

Reply:
We are not using this feature for the time being. Can be added afterwards.